quickstarts/microsoft.web/web-app-sql-database/azuredeploy.json

```diff
@@ -168,9 +168,8 @@
         "displayName": "AppInsightsComponent"
       },
       "properties": {
-        "ApplicationId": "[variables('webSiteName')]",
```

Comment:
how did we confirm this is not needed?

Reply:
https://docs.microsoft.com/en-us/azure/templates/microsoft.insights/components?tabs=json
As per documentation, there is no field for "ApplicationId". Instead, we need to provide "Application_Type"